test/e2e/rte.go

```diff
@@ -81,4 +89,32 @@ var _ = ginkgo.Describe("[RTE] Node topology updater", func() {
 			}, time.Minute, 5*time.Second).Should(gomega.BeTrue())
 		})
 	})
+
+	ginkgo.AfterEach(func() {
+		err := f.ClientSet.CoreV1().ConfigMaps("default").Delete(context.TODO(), TopologyUpdaterConfigMapName, metav1.DeleteOptions{})
```

Comment:
the namespace looks suspicious - we don't create the configmap in the "default" namespace